docs/references.bib

```diff
@@ -4216,3 +4216,198 @@ @incollection{ecwmf_physical_2016
+  journal = {Journal of Hydrology},
+  volume = {608},
+  pages = {127574},
+  year = {2022},
+  issn = {0022-1694},
+  doi = {https://doi.org/10.1016/j.jhydrol.2022.127574},
+  url = {https://www.sciencedirect.com/science/article/pii/S0022169422001494},
+  author = {Esteban Alonso-González and Jesús Revuelto and Steven R. Fassnacht and Juan
+    {Ignacio López-Moreno}},
+  abstract = {The duration of the seasonal snowpack determines numerous aspects of the
+    water cycle, ecology and the economy in cold and mountainous regions, and is a balance
+    between the magnitude of accumulated snow and the rate of melt. The contribution of
+    each component has not been well quantified under contrasting topography and
+    climatological conditions although this may provide useful insights into how snow cover
+    duration could respond to climate change. Here, we examined the contribution of the
+    annual peak snow water equivalent (SWE) and the seasonal melt rate to define the
+    duration of the snowpack over temperate mountains, using snow data for mountain areas
+    with different climatological characteristics across the Iberian Peninsula. We used a
+    daily snowpack database for the period 1980--2014 over Iberia to derive the seasonal
+    peak SWE, melt rate and season snow cover duration. The influence of peak SWE and melt
+    rates on seasonal snow cover duration was estimated using a stepwise linear model
+    approach. The stepwise linear models showed high R-adjusted values (average R-adjusted
+    = 0.7), without any clear dependence on the elevation or geographical location. In
+    general, the peak SWE influenced the snow cover duration over all of the mountain areas
+    analysed to a greater extent than the melt rates (89.1\%, 89.2\%, 81.6\%, 93.2\% and
+    95.5\% in the areas for the Cantabrian, Central, Iberian, Pyrenees and Sierra Nevada
+    mountain ranges, respectively). At these colder sites, the melt season occurs mostly in
+    the spring and tends to occur very fast. In contrast, the areas where the melt rates
+    dominated snow cover duration were located systematically at lower elevations, due to
+    the high interannual variability in the occurrence of annual peak SWE (in winter or
+    early spring), yielding highly variable melt rates. However, in colder sites the melt
+    season occurs mostly in spring and it is very fast in most of the years. The results
+    highlight the control that the seasonal precipitation patterns, in combination with
+    temperature, exert on the seasonal snow cover duration by influencing the peak SWE and
+    suggest a future increased importance of melt rates as temperatures increase. Despite
+    the high climatological variability of the Iberian mountain ranges, the results showed
+    a consistent behaviour along the different mountain ranges, indicating that the methods
+    and results may be transferrable to other temperate mountain areas of the world.}
```

Comment:
Do we need the full abstract here ? For a smaller file and for coherence with the other entires, I think we could remove the abstracts here.